Api/RestCustomerInterface.php

```diff
@@ -70,4 +70,21 @@ public function deleteCertificate(
         $scopeId = null,
         $scopeType = \Magento\Store\Model\ScopeInterface::SCOPE_STORE
     );
+
+    /**
+     * Perform REST request to update a customer.
+     *
+     * @param \Magento\Customer\Api\Data\CustomerInterface $customer
+     * @param bool|null $isProduction
+     * @param string|int|null $scopeId
+     * @param string $scopeType
+     * @return mixed
```

Comment:
Please specify your return type when it's known. Only use `mixed` whenever you do not know what the return type will be (which should also never be the case)

```suggestion
     * @return \Magento\Framework\DataObject
```